vault/external_tests/upgrade_testing/upgrade_testing_binary/upgrade_test.go

```diff
@@ -0,0 +1,67 @@
+package upgrade_testing_binary
+
+import (
+	"os"
+	"testing"
+
+	"github.com/hashicorp/vault/sdk/helper/testcluster"
+	"github.com/hashicorp/vault/sdk/helper/testcluster/docker"
+)
+
+func DockerPrivilegedNamespaceClient(t *testing.T, nodeCount int) *docker.DockerCluster {
```

Comment:
I think this wasn't meant to be committed

Reply:
Oops again!! Removed this.